Jenkinsfile

```diff
@@ -0,0 +1,168 @@
+pipeline {
+    agent {
+        label 'llrgrhgtrig.in2p3.fr'
+    }
+    environment {
+        EMAIL_TO = 'jenkins@llr.in2p3.fr'
+    }
+    options {
+        skipDefaultCheckout() 
+    }
+    stages {
+        stage('InstallAutoValidationPackage') {
+            steps {
+                echo 'Install automatic validation package HGCTPGValidation.'
+                sh '''
+                uname -a
+                whoami
+                pwd
+                ls -l
+                if [ -d "./HGCTPGValidation" ] 
+                then
+                    rm -rf HGCTPGValidation
+                fi
+                git clone -b Jenkins-developments https://github.com/hgc-tpg/HGCTPGValidation HGCTPGValidation
+                ~/grid_login
```

Comment:
`grid_login` is called in multiple places. Is it needed to call it even in jobs where grid data are not fetched?

Reply:
The grid_login was added for accessing grid data. If we plan to use only local data files, it is not needed.